CredentialProvider.Microsoft/Resources.resx

```diff
@@ -459,4 +459,17 @@ NuGet workarounds
   <data name="UIFlowStarted" xml:space="preserve">
     <value>Using the {0}  flow for uri {1}. User sign-in required in a pop-up authentication window.</value>
   </data>
+  <data name="HostAccepted" xml:space="preserve">
+    <value>Host accepted: {0}</value>
+  </data>
+  <data name="OnPremDetected" xml:space="preserve">
+    <value>Detected an on premise Azure DevOps Server. Please use a PAT and VSS_NUGET_EXTERNAL_FEED_ENDPOINTS variable to use this credential provider.</value>

```

Comment:
```suggestion
    <value>Detected on-premises Azure DevOps Server.</value>
```
I don't think we need to tell customers about weird environment variables here. Customers should use regular nuget credential config mechanisms to set up their PATs

Reply:
With Aurelie's Connect-to-feed changes, I don't think we need to add any specific instructions here at all.